proposals/crd-support.md

```diff
@@ -0,0 +1,52 @@
+# Better support for CRDs
+
+## Summary
+This is an outline for how CRDs can be handled in Helm. There are at least two ways to do this currently, but both have issues. With the implementation suggested here, CRDs are getting special treatment to make sure that they are installed correctly and are updated and deleted in a safe way. CRDs will be managed as part of a release, but will be orphaned rather than deleted when a release is removed. Similar, CRDs will be adopted into a release if they already exist.
+
+## Motivation
+The current ways to handle CRDs in Helm have issues and these are known to have caused real problems for users. The goal here is to handle CRDs in a way that is intuitive and flexible for chart developers, as well as easy to use and safe for helm users. The goal is that this could be introduced in Helm 2 as well as make it into Helm 3.
```

Comment:
Is there a documentation somewhere that points to the current ways to handle CRDs in Helm? If not, can that be included here for completeness sake?

Reply:
I added a link to an issue that has a great description of the current ways to handle CRDs with Helm and the problems with them.